examples/dataflow-data-generator/data-generator-pipeline/data_generator/ParquetUtil.py

```diff
@@ -25,30 +25,56 @@ def get_pyarrow_translated_schema(string_schema):
     :param string_schema:
     :return: pyarrow schema
     """
-    type_conversions = {
-        'STRING': pa.string(),
-        'NUMERIC': pa.int64(),
-        'BYTE': None,
-        'INTEGER': pa.int64(),
-        'FLOAT': pa.float64(),
-        'NUMERIC': pa.int64(),
-        'BOOLEAN': pa.bool_(),
-        'TIMESTAMP': pa.timestamp('us'),
-        'DATE': pa.date32(),
-        'TIME': pa.time64('us'),
-        'DATETIME': pa.timestamp('us'),
-        'GEOGRAPHY': None,
-        'RECORD': None
-    }
+    def _bq_to_pa_type(field):
+        """
+        A function to convert BigQuery types to pyarrow types.
+        :param field (bigquery.schema.SchemaField)
+        :return: pa.DataType
+        """
+        type_conversions = {
+            'STRING': pa.string(),
+            'NUMERIC': pa.int64(),
+            'BYTE': None,
+            'INTEGER': pa.int64(),
+            'FLOAT': pa.float64(),
+            'NUMERIC': pa.int64(),
```

Comment:
This is totally my fault, but I listed NUMERIC twice. Would you mind deleting this one?

Reply:
removed.